content/blog/configuring-azure-ad-with-greenlake-cloud-platform-and-aruba-central.md

```diff
@@ -0,0 +1,236 @@
+
+<img src="/img/0b085a5aef05404e9ecdf52cb9088feb/ccs-saml-config-settings-summary.png" alt="saml-settings" height="70%" width="70%">
+
+* Create the recovery user per the instructions
+* Validate the settings are correct
+* Save and Finish the configruation.
+* If you get an error that the SAML configuraiton wasn't completed using the account with the @domain.com. You'll have to log back up and login with the SAML domain and go through the above configuration again.
+
+## Login to GLCP and Aruba Central using Azure AD
+
+* Once you've completed the above steps, login to central using your Azure AD email.
+
+<img src="/img/0b085a5aef05404e9ecdf52cb9088feb/ccs_login.png" alt="ccs_login" height="40%" width="40%"><img src="/img/0b085a5aef05404e9ecdf52cb9088feb/ccs_login_saml.png" alt="ccs_login_saml" height="40%" width="40%">
+
+* If everything is working correctly, you should have logged into GLCP and Aruba Central is an option to Launch.
+
+## Using Azure AD MFA
+* By default, Azure AD enables MFA. However, for testing and demos, it's much easier to disable MFA on your accounts. To disable MFA, please see the following documentation: [What are security defaults](https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/concept-fundamentals-security-defaults)
+
+## Troubleshooting
+* There's a useful 3rd party browser tool called: SAML Tracer
+* This tool will allow you to verify the attributes you're sending to Central.
+* It can be useful when configuratin SAML with multiple Central accounts or domains
+* SAML Tracer
+[Chrome](https://chrome.google.com/webstore/detail/saml-tracer/mpdajninpobndbfcldcmbpnnbhibjmch?hl=en)
+[FireFox](https://addons.mozilla.org/en-US/firefox/addon/saml-tracer/)
+
+![Image](/img/0b085a5aef05404e9ecdf52cb9088feb/firefox-saml-tracer.png)
+
+
+
+
+
+## Appendix: Generating the `hpe_ccs_attribute`
+
+The `hpe_ccs_attribute` is used to determine your GLCP account.  The format for the `hpe_ccs_attribute` is as follows:
+
+
+<img src="/img/0b085a5aef05404e9ecdf52cb9088feb/hpe_ccs_attribute-img1.png" alt="hpe_ccs_attribute-img1" height="75%" width="75%">
+
+
+An Example `hpe_ccs_attribute` for a single GLCP and Aruba Central account would be:
+
+```
+version_1#2fd5f97acbc211ecadc006baf610dd36:00000000-0000-0000-0000-000000000000:Account Administrator:ALL_SCOPES:683da368-66cb-4ee7-90a9-ec1964768092:Aruba Central Administrator:ALL_SCOPES
+```
+
+or
+
+
+```
+version_1#5b0ec0e8b4f411eca432ba72799953ac:00000000-0000-0000-0000-000000000000:Account Administrator:ALL_SCOPES:683da368-66cb-4ee7-90a9-ec1964768092:Aruba Central Administrator:ALL_SCOPES#5b0ec0e8b4f411eca432ba72799953ac:00000000-0000-0000-0000-000000000000:Account Administrator:ALL_SCOPES
+```
+
+
+If you're a Managed Service Provider (MSP), then the `hpe_ccs_attribute` for Administrator rights to GLCP and Aruba Central for all customer tenant accounts:
```

Comment:
Add HPE prior to GLCP.
You're missing a verb here. I suspect it's "is" and should be placed just prior to the colon.